api/admin/controller.py

```diff
@@ -1068,6 +1116,15 @@ def admin_auth_services(self):
                         domains=json.loads(auth_service.external_integration.setting("domains").value),
                     )
                 ]
+            elif auth_service and auth_service.provider == AdminAuthenticationService.LOCAL_PASSWORD:
+                admins = self._db.query(Admin).filter(Admin.password_hashed != None)
+                auth_services = [
+                    dict(
+                        name=auth_service.name,
+                        provider=auth_service.provider,
+                        admins=[dict(email=admin.email) for admin in admins],
```

Comment:
When you edit the password service in the admin interface, you see the emails of current admins and you can can delete them from the list.